src/test/java/org/apache/maven/plugins/javadoc/JavadocReportTest.java

```diff
@@ -594,27 +595,6 @@ public void testOptionsUmlautEncoding() throws Exception {
         }
     }
 
-    /**
-     * @throws Exception if any
-     */
-    public void testExceptions() throws Exception {
-        try {
-            Path testPom = unit.resolve("default-configuration/exception-test-plugin-config.xml");
-            JavadocReport mojo = lookupMojo(testPom);
```

Comment:
This lookup fails and throws an exception, causing the test to pass. It never executes the mojo which is what it's supposed to test.

Reply:
Maybe we should catch MojoExecutionException and check the root cous of the failing test instead of the generic Exception?

Reply:
No, the execute method is never invoked. The lookupMojo method fails because the mojo was poorly written. It's been months since I wrote this, so I'm not sure if the test case was always broken or the code itself, but this is trying to test line 604 but instead the exception is thrown before we get there. The test fails if you catch MojoExecutionException instead of Exception.